templates/cloudfront-site.yaml

```diff
@@ -97,11 +99,14 @@ Resources:
         - Key: Solution
           Value: ACFS3
 
-  CloudFrontOriginAccessIdentity:
-    Type: AWS::CloudFront::CloudFrontOriginAccessIdentity
+  OriginAccessControl:
+    Type: AWS::CloudFront::OriginAccessControl
     Properties:
-      CloudFrontOriginAccessIdentityConfig:
-        Comment: !Sub 'CloudFront OAI for ${SubDomain}.${DomainName}'
+      OriginAccessControlConfig:
+        Name: !Sub 'oac-${AWS::StackName}'
```

Comment:
You may want to include the region as well, since the OAC Name is global but stack names are scoped to a region. You can see an example of where this was a problem in https://github.com/awslabs/amazon-s3-find-and-forget/commit/efc6889fc8359f9313e60f548c51579e015825de

I know this sample is currently authored to only support deploys in us-east-1, so the benefit would mostly be if that support was ever expanded, or for anyone cribbing/copying it as an example to their own templates.

Reply:
Good idea. I've added that.